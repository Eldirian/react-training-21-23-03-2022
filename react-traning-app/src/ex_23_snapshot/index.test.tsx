import renderer from 'react-test-renderer';
import { Alert } from '.';

describe('Alert', () => {
    test('renders correctly as error', () => {
        const tree = renderer.create(<Alert type='error' text='Lorem ipsum' />);
        expect(tree).toMatchSnapshot();
    })
    test('renders correctly as success', () => {
        const tree = renderer.create(<Alert type='success' text='Lorem ipsum' />);
        expect(tree).toMatchSnapshot();
    })
    test('renders correctly as notice', () => {
        const tree = renderer.create(<Alert type='notice' text='Lorem ipsum' />);
        expect(tree).toMatchSnapshot();
    })
})