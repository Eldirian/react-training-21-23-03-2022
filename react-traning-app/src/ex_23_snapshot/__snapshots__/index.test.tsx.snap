// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly as error 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
    }
  }
>
  Lorem ipsum
</div>
`;

exports[`Alert renders correctly as notice 1`] = `
<div
  style={
    Object {
      "backgroundColor": "lightblue",
    }
  }
>
  Lorem ipsum
</div>
`;

exports[`Alert renders correctly as success 1`] = `
<div
  style={
    Object {
      "backgroundColor": "green",
    }
  }
>
  Lorem ipsum
</div>
`;
